src/vivarium_testing_utils/automated_validation/comparison.py

```diff
@@ -164,6 +164,12 @@ def get_frame(
         test_proportion_data, reference_data = self._align_datasets(stratifications)
 
         test_proportion_data = test_proportion_data.rename(columns={"value": "rate"}).dropna()
+        # Reference data can be a float or dataframe
```

Comment:
@patricktnast Im not sure if this is the best way or best sport to handle this dataframe or float issue we are going to run into but I had to add this so mypy  would pass. I think we should think about this more carefully with the reset of the implementation for this ticket.

Reply:
I moved this now after thinking about it more.